pkg/common/common.go

```diff
@@ -216,3 +217,14 @@ func StripDefaultPorts(fromUrl string) (string, error) {
 	}
 	return r.URL.String(), nil
 }
+
+// GetImagePullPolicy get imagePullPolicy for a container, based on its image.
+// If image contains a digest, use IfNotPresent; otherwise, Always.
+// Reference: https://github.com/distribution/distribution/blob/v2.7.1/reference/reference.go
```

Comment:
reference I found good https://stackoverflow.com/questions/37861791/how-are-docker-image-names-parsed

Reply:
I think better source of reference is here:
   https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests 
   
   Based on that @sha256 is fine, maybe another validation if the followed set of characters is exactly 64 ?
   
   https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256
   
   Also would be wise to include sha512, because it's part of the spec ?

Reply:
like the reference

will wait feedback about length check to update both, if agreed

Reply:
added length and sha512 checks